devel-common/src/tests_common/test_utils/db.py

```diff
@@ -352,5 +354,5 @@ def clear_all():
     clear_db_connections(add_default_connections_back=True)
     clear_db_deadline()
     clear_dag_specific_permissions()
-    if AIRFLOW_V_3_0_PLUS:
-        clear_db_dag_bundles()
+    clear_db_dag_parsing_requests()
```

Comment:
```python
if AIRFLOW_V_3_0_PLUS:
    clear_db_backfills()
    clear_db_dag_bundles()
    clear_db_dag_parsing_requests()
```

Reply:
Yep thanks, just saw the CI failing on compat checks.

Reply:
Done thanks